src/routes/Apps/App/TrainDialogs.tsx

```diff
@@ -305,11 +305,6 @@ class TrainDialogs extends React.Component<Props, ComponentState> {
             }
         }
 
-        // TODO: Handle reload when actions not yet loaded
```

Comment:
Don't remember.  My guess would be that it refers to page reload (i.e. browser) and prevents a null ref.  I'd be hesitant to take it out unless it is causing other issues.

Reply:
It is causing other issues which why I am asking about it.  I think the alternative is to prevent the server from creating the dialog.  I think there was mechanism to show train dialogs even though the user never saved it to preserve their work hoping they could finish it later.

There is mismatch in that when creating dialog manually we require an action to save, but server does not require an action, this then makes the dialog un openable.